docs/src/components/PropsTableTooltip.tsx

```diff
@@ -0,0 +1,98 @@
+'use client';
+import { Tooltip } from '@base_ui/react/Tooltip';
+import * as React from 'react';
+import { ToolbarButton } from './ToolbarButton';
+import { Popup } from './Popup';
+
+export function PropsTableTooltip({ children }: React.PropsWithChildren) {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    // TODO: this was more effort than I expected, we need to chat about popovers that open on hover
+    <Tooltip.Root
```

Comment:
Curious what happened here? Is it just that `<Tooltip.Provider>` doesn't exist for popovers? This current pattern isn't accessible unlike the `<Popover.Root openOnHover>` one

Reply:
Yeah I have a point to chat through this on Monday, going to be easier to show.

For now I made it so that the interactions feel good and SR will read out the prop description when entering the cell as if it was regular text.

Reply:
Reverted to popovers that open on click while we figure it out